purchasing/opportunities/front/views.py

```diff
@@ -291,6 +291,21 @@ def browse():
         upcoming=sorted(upcoming, key=lambda i: i.planned_submission_start),
     )
 
+@blueprint.route('/opportunities/expired', methods=['GET'])
+def expired():
+    '''View expired contracts
+    '''
+    expired = Opportunity.query.filter(
+        Opportunity.planned_submission_end < datetime.date.today(),
+        Opportunity.is_public == True
+    ).all()
+
+    current_app.logger.info('BEACON FRONT OPPORTUNITY BROWSE EXPIRED VIEW')
```

Comment:
Can you change this to be more in line with other logger strings? The idea is to have a single short string that is easy to grep/search for.